### ISSUE TITLE
Configurable alarm auto-snooze

### ISSUE THREAD

**Issue by eandersons** [you-apps/ClockYou#220]
### Describe your suggested feature

It would be nice for alarms to have a configurable auto-snooze.
Once an alarm goes off, it automatically snoozes after a specified time for a specified interval and repeats a specified amount of times.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by M00NJ** [comment]
That doesn't make any sence to me. An alarm is supposed to wake you up and should require a conscious decision to either turn it off or have it repeated. If it automatically snoozes itself you might as well set it when you actually want to get up. That sound more like you want an audible reminder that you should get up soon. I can't really relate here, but in that case I'd just use a calendar app to get a notification sound. What clock app are you using that has this feature?

**Comment by eandersons** [comment]
I have seen this kind of functionality in some clock application, but it was a while ago and I do not recall which one it was (probably some custom ROM's clock app).
After a quick web search seems that, for example, Alarm Clock Plus and Alarm Clock Xtreme,  have an auto snooze functionality.

The reasoning for this request is that I have a flexible work hours, therefore each evening I set a time in the next morning that I should wake up at, but if at that time I do not feel like waking up, I wait for an alarm to end (as I leave phone in the other room) and sleep a bit more till the next alarm. Therefore it is kind of pre-made conscious decision.
Setting reminders in calendar is tedious. For now I am setting multiple alarms in the stock clock app that are set up to ring for 5 minutes.
Currently I am not using Clock You anyway as alarms are without sound - it ringed only once and there have been silence from Clock You alarm since then.

In the end it is all about choice - the default behaviour for a Clock You alarm should be to ring until it is turned off, but configurable options would be a nice to have feature to adjust alarms to one's needs and waking-up habits.

**Comment by M00NJ** [comment]
This is a very niche feature, so we won't work on this ourselfs. A PR for this feature might get accepted though. Off topic, but imo the whole "I don't feel like getting up" thing is not a good habit, so looking for features that enable this behavior doesn't seem helpful to me. The goal should be to set the timer for when you need to get up and actually follow through with it. 

About ClockYou not playing alarm sounds: There are no open issues for this and I cannot reproduce it. Please make sure you are using the latest version, have the switch for each alarm turned on, have the alarm volume on your device set to and audible level and give the app all needed permissions to work properly.

**Comment by SuhasDissa** [comment]
An auto-snooze feature can be very useful. Ideally, after a specific period of time (e.g., 10 minutes), the alarm should automatically stop ringing.
This prevents the alarm from continuously ringing all day and draining the battery if the phone is left unattended at home or in a bag.

**Comment by M00NJ** [comment]
A pref to set the alarm ringing duration would do that too. But feel free to reopen this, if it's useful to you.

**Comment by brckd** [comment]
i just came back from vacation and the alarm on my tablet has ringed for about 24 hours, draining 23% battery out of 11200 mAh.
i mean thas not too bad, but stopping the timer after maybe three hours could protect the batteries and my neighbours during longer vacations
